### PR TITLE
Mostly harmless

### PR DESCRIPTION
**Pull request by simogasp**
Some formatting, documentation and renaming :-)

### CODE REVIEW

File: src/openMVG/matching/regions_matcher.hpp [hunk 1]
```diff
@@ -32,61 +43,98 @@ void DistanceRatioMatch
...
-   */
-  virtual bool Match
-  (
-    const float f_dist_ratio,
-    const features::Regions& query_regions,
-    matching::IndMatches & vec_putative_matches
-  ) =0;
+    
+   /**
+    * @brief The empty constructor.
```
Comment (pmoulon) [inline src/openMVG/matching/regions_matcher.hpp]: destructor

File: src/openMVG/matching/regions_matcher.hpp [hunk 2]
```diff
@@ -132,7 +194,13 @@ class RegionsMatcherT : public RegionsMatcher
   }
 
   /**
-   * @brief Match some regions to the database of internal regions.
+   * @brief
```
Comment (pmoulon) [inline src/openMVG/matching/regions_matcher.hpp]: Empty brief?

File: src/openMVG/matching/regions_matcher.hpp (hunk 1 above)
Comment (simogasp) [inline src/openMVG/matching/regions_matcher.hpp]: e29b736

File: src/openMVG/matching/regions_matcher.hpp (hunk 2 above)
Comment (simogasp) [inline src/openMVG/matching/regions_matcher.hpp]: e29b736

File: src/openMVG/sfm/pipelines/global/GlobalSfM_rotation_averaging.cpp [hunk 3]
```diff
@@ -53,9 +53,12 @@ bool GlobalSfM_Rotation_AveragingSolver::Run(
         return false;
       KeepOnlyReferencedElement(set_remainingIds, relativeRotations);
     }
-    break;
+	break;
+    default:
+    std::cerr << "Unknown relative rotation inference method: " 
+            << (int) eRelativeRotationInferenceMethod << std::endl;
```
Comment (simogasp) [inline src/openMVG/sfm/pipelines/global/GlobalSfM_rotation_averaging.cpp]: @pmoulon and maybe a `return false;` after the cerr?

Comment (pmoulon) [inline src/openMVG/sfm/pipelines/global/GlobalSfM_rotation_averaging.cpp]: Thanks for the suggestion.
Note that bSuccess will be false and false will be return later

**Pull request closed without merging**